src/ui/sub_views/details_pane.rs

```diff
@@ -61,11 +63,20 @@ impl DetailsPane
         c.x -= 1;
         c.width += 2;
         c.height += 1;
+        let vertical_chunks: Vec<Rect> = if ClientThreadId::try_from(&request.request_msg).is_ok() {
+            Layout::default()
+                .direction(Direction::Vertical)
+                .margin(0)
+                .constraints([Constraint::Percentage(80), Constraint::Percentage(20)].as_ref())
+                .split(block.inner(c))
+        } else {
+            Vec::from([block.inner(c)])
+        };
```

Comment:
At some point we should have some kind of a kill switch (either implicit or explicit) for the call stack mechanism.

It's not a problem if we just show the client-reported values, but once we start capturing the actual call stack stack frames by pausing the process, we should be careful we don't allow arbitrary processes to tell Proxide to pause (and capture the stack trace) from other processes. This is especially important, if Proxide binds itself to public network interface and allows connections from non-local clients. I think it might be sensible to enable the call stack capture only when Proxide is bound to a local-only interface (at least by default).